src/dodal/devices/attenuator.py

```diff
@@ -17,7 +17,7 @@ class Attenuator(Device):
     def set(self, transmission) -> SubscriptionStatus:
         """Get desired states and calculated states, return a status which is complete once they are equal"""
 
-        LOGGER.info("Using current energy")
+        LOGGER.info("Setting current energy ")
```

Comment:
Should: I think the previous was more correct. "Setting current energy" makes it sound like we're changing energy of the beamline

Reply:
Thanks, I've changed these now